src/DevChatter.Bot.Core/Games/DealNoDeal/DNDCommand.cs

```diff
@@ -0,0 +1,54 @@
+using DevChatter.Bot.Core.Commands;
+using DevChatter.Bot.Core.Commands.Operations;
+using DevChatter.Bot.Core.Data;
+using DevChatter.Bot.Core.Data.Model;
+using DevChatter.Bot.Core.Events.Args;
+using DevChatter.Bot.Core.Systems.Chat;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DevChatter.Bot.Core.Games.DealNoDeal
+{
+    public class DNDCommand : BaseCommand, IGameCommand
```

Comment:
This breaks the C# convention for capitalization, see https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers

Maybe even call it `DealOrNoDealCommand`?